.github/workflows/main.yml

```diff
@@ -43,6 +43,13 @@ jobs:
         run: |
           python setup.py install --user
 
+      # This needs to happen *after* installing pywin32 since
+      # AutoDuck/py2d.py currently relies on runtime imports for introspection
+      # This isn't included in the wheel (TODO: could we?)
+      # and only servces as a PR test for the docs.yaml workflow
+      - name: Generate PyWin32.chm help file
+        run: python AutoDuck/make.py
+
```

Comment:
If we merged both workflows, we could push these as artifacts, then no need to download from PyPI